src/logging.cc

```diff
@@ -191,7 +191,7 @@ GLOG_DEFINE_string(log_backtrace_at, "",
 #include <BaseTsd.h>
 #define ssize_t SSIZE_T
 #endif
-static ssize_t pread(int fd, void* buf, size_t count, off_t offset) {
+static ssize_t pread(int fd, void* buf, unsigned int count, off_t offset) {
```

Comment:
Ok. I changed it to size_t again and added a cast for read which uses unsigned int.